deployments/roman/image/Dockerfile

```diff
@@ -70,6 +46,62 @@ RUN cd /tmp && \
     make && \
     make install
 
+# ------------------------------------------------------------------------------
+# Install environments
+
+# All specs for frozen builds need to be available before their normal installs
+USER ${NB_UID}
+
+COPY --chown=${NB_UID}:${NB_GID} environments/frozen/  /opt/environments/frozen
+
+# --- # ------------------------------------------------------------------------------
+# --- # MIRAGE
+# --- # RUN /opt/common-scripts/env-create  mirage /opt/environments/mirage/mirage.yml
+# --- RUN /opt/common-scripts/env-clone base mirage
+
+# --- COPY --chown=${NB_UID}:${NB_GID} environments/mirage/mirage.pip /opt/environments/mirage/mirage.pip
+# --- RUN /opt/common-scripts/env-update  mirage /opt/environments/mirage/mirage.pip
+
+# --- RUN /opt/common-scripts/install-common mirage
+
+# ------------------------------------------------------------------------------
+# CVT
+RUN /opt/common-scripts/env-clone  base   cvt
+
+COPY --chown=${NB_UID}:${NB_GID} environments/cvt/cvt.yml /opt/environments/cvt.yml
+RUN /opt/common-scripts/env-create  cvt /opt/environments/cvt/cvt.yml
+
+RUN /opt/common-scripts/install-common cvt
+
+# ----------------------------------------------------------------------
+# Roman CAL
+# RUN /opt/common-scripts/env-clone base roman-cal   # too many conflicts
+
+COPY --chown=${NB_UID}:${NB_GID} environments/roman-cal/roman-cal.yml /opt/environments/roman-cal/roman-cal.yml
+RUN /opt/common-scripts/env-create  roman-cal /opt/environments/roman-cal/roman-cal.yml
+
+COPY --chown=${NB_UID}:${NB_GID} environments/roman-cal/roman-cal.pip /opt/environments/roman-cal/roman-cal.pip
+RUN /opt/common-scripts/env-update  roman-cal /opt/environments/roman-cal/roman-cal.pip
+
+COPY --chown=${NB_UID}:${NB_GID} environments/roman-cal/octarine.pip /opt/environments/roman-cal/octarine.pip
+RUN /opt/common-scripts/env-update  roman-cal /opt/environments/roman-cal/octarine.pip
+
+RUN /opt/common-scripts/install-common roman-cal
+
+# ------------------------------------------------------------------------------
+# All environments files,  including test files
+COPY --chown=${NB_UID}:${NB_GID} environments/  /opt/environments
+
+# ----------------------------------------------------------------------
+# Run the hook script for updates which are common to both the standalone
+# image and to hub cloud operation.
+#
+USER root
+RUN chown -R ${NB_UID}:${NB_GID}  /home/${NB_USER}
+
+USER ${NB_UID}
+RUN /opt/environments/post-start-hook
```

Comment:
This really shouldn't be here anymore,  post-start-hook needs to be called from Dockerfile.trailer so now is.

Reply:
Fortunately,  it's just the Dockerfile which needs to be regenerated,  Dockerfile.custom has already been updated.